app/assets/config/manifest.js

```diff
@@ -1,3 +1,4 @@
 //= link_tree ../images
 //= link_directory ../stylesheets .css
 //= link_tree ../builds
```

Comment:
As the previous code reviewer suggested kindly fix this error:

![Screenshot 2023-03-25 at 1 31 25 PM](https://user-images.githubusercontent.com/79506105/227714920-7896a585-2add-4345-ab29-dcccc1a611b8.png)